File: CT_BAEKJOON_2753.py
#https://www.acmicpc.net/problem/2753
#https://www.acmicpc.net/source/41979054

a = input()
a = int(a)


if (a%4==0 and a%100 != 0):
    print(1)
elif a%400==0:
    print(1)
else:
    print(0)

File: CT_BAEKJOON_2753.c
//https://www.acmicpc.net/problem/2753
//https://www.acmicpc.net/source/42091980

#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main()
{
   int years;

   
   scanf("%d", &years);

   if (years % 4 == 0 && years % 100 != 0 || years % 400 == 0)
   {
      printf("1");
   }
   else printf("0");

   return 0;
}